src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -622,8 +622,9 @@ private void createAndSetStatistics()
             }
             else
             {
-                final int length = citizen.getJob().split("\\.").length;
-                final String job = citizen.getJob().split("\\.")[length - 1].toLowerCase(Locale.ENGLISH);
+                final String[] splitString = citizen.getJob().split("\\.");
```

Comment:
This change is irrelevant, but still...… The JVM will optimize this away during runtime,